### PR TITLE
Implementing Countable.js to count words.

### PR DESCRIPTION
**Pull request by nikhilgupta1211**
#696 
Changed the words_count method to work with countable.js.

### CODE REVIEW

File: app/assets/javascripts/Countable.js [hunk 1]
```diff
@@ -0,0 +1,407 @@
+/**
```
Comment (differentreality) [inline app/assets/javascripts/Countable.js]: This is not how we add new js in the app. Please use the respective gem.

**Comment by differentreality** [comment]
Excellent, that looks much better now @nikhilgupta1211 and I will review it soon.

**Comment by nikhilgupta1211** [comment]
done :)

**Comment by differentreality** [comment]
This looks good to me. @ChrisBr ?

**Comment by differentreality** [comment]
:+1: Good work @nikhilgupta1211 Please squash and rebase for merging.

**Comment by nikhilgupta1211** [comment]
done :)

**Pull request merged**